### DISCUSSION TITLE
Application Icons

### DISCUSSION THREAD

**Discussion by vincode-io** [vincode-io/Zavala#222] [General]
Every once in a while I get a request for multiple application icons. I think multiple icons are fun and can have practical uses when decorating your Home Screen or Desktop. Unfortunately, Zavala will probably never support them. Here's why.

Most open source projects like Zavala never get help from a good designer. I was fortunate enough that Brad Ellis took an interest in the project and helped me out. Things like how the tags get added in the Editor are his design. He helped me refine the app UI. He also did not one, but two, amazing icons for Zavala. Both convey what Zavala is while still conforming to the different design languages of macOS and iOS.

To get two professionally done icons for a small open source project is amazing. You should see the one's I've created myself for other side projects. Not good. I'm just grateful for what I got and am not comfortable asking for more from Brad.

I also am not comfortable customizing his designs. He understands how to work shape and color in ways I will never understand. Degrading his work by applying my hand to it is disrespectful in my opinion.

If I charged for Zavala, I could pay Brad what he is worth to make more icons. I just think there are too many advantages to keeping Zavala free to ever compromise them. So, sorry. Most likely multiple application icons will never happen.

**Comment by smellythief** [comment]
Have you thought about making it an IAP? I commend you for keeping it free, but for something like this that is entirely aesthetic, and not even noticeable while using the app, an IAP may be a nice way to allow people to tip you. And allow you to pay Brad.
